### PR TITLE
Step 6.2 Develop backend (API) part.

### PR DESCRIPTION
**Pull request by akimov228aleksei**
- Created fixtures.
- Added quick start in README.
- Updated DB settings.

### CODE REVIEW

File: src/management/Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+run_server:
+	python -m pip install -U pip
```
Comment (jimmyhollywood) [inline src/management/Makefile:2]: Please, fix the formatting

File: src/management/Makefile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	find . -path "*/migrations/*.py" -not -name "__init__.py" -delete
+		find . -path "*/migrations/*.pyc"  -delete
+			python ./manage.py makemigrations
+				python ./manage.py migrate
+
+reset_sqlite:
+	find . -path "*/migrations/*.py" -not -name "__init__.py" -delete
+		find . -path "*/migrations/*.pyc"  -delete
+			rm db.sqlite3
+				python ./manage.py makemigrations
```
Comment (jimmyhollywood) [inline src/management/Makefile:22]: One of the features of using Makefiles is that you can combine multiple commands -- here two latest lines seems like a duplication of `run_migration` commands - so you can just call that command here

Comment (jimmyhollywood) [reply src/management/Makefile:22]: And just wondering -- what the purpose of this command?

File: README.md [hunk 3]
```diff
@@ -3,4 +3,9 @@
 [![Python-package Actions Status](https://github.com/akimov228aleksei/DRF/workflows/Coverage/badge.svg?branch=dev)](https://github.com/akimov228aleksei/DRF/actions)
 [![Coverage Status](https://coveralls.io/repos/github/akimov228aleksei/DRF/badge.svg?branch=dev)](https://coveralls.io/github/akimov228aleksei/DRF?branch=dev)
 
-## The application consists of two parts - frontend and backend. The backend contains an API for working with the database. Frontend contains an interface for communicating with the API.
\ No newline at end of file
+## The application consists of two parts - frontend and backend. The backend contains an API for working with the database. Frontend contains an interface for communicating with the API.
+
+## Quick start
+    1. Define invironment variables for DB (or use default variables)
```
Comment (jimmyhollywood) [inline README.md:9]: Describe here the list of the env variables

File: src/management/Makefile [hunk 4]
```diff
@@ -0,0 +1,23 @@
+run_server:
```
Comment (jimmyhollywood) [inline src/management/Makefile:1]: This looks a bit confusing for me -- the command name is `run_server` but it does a lot work outside the scope based on the title.
`run_server` have to simply run the web server =)
I think it is reasonable to add a separate command to initialize the application. Let say `init_app` command should install the python packages, run migration and load fixtures

**Review by jimmyhollywood — CHANGES_REQUESTED** [review]

File: src/management/Makefile (hunk 1 above)
Comment (akimov228aleksei) [reply src/management/Makefile:2]: Fixed

File: src/management/Makefile (hunk 2 above)
Comment (akimov228aleksei) [reply src/management/Makefile:22]: `reset_sqlite` is command which repeat `reset_db` command, but `reset_sqlite` also delete sqlite.db file. It is helpful when you use sqlite3 database and want to reset db.

File: src/management/Makefile (hunk 4 above)
Comment (akimov228aleksei) [reply src/management/Makefile:1]: Fixed.

File: README.md (hunk 3 above)
Comment (akimov228aleksei) [reply README.md:9]: Fixed.

**Review by jimmyhollywood — APPROVED** [review]

**Pull request merged**